contract/scripts/deploy-contract.js

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env node
+/* global process, fetch, setTimeout */
+// @ts-check
+import '@endo/init';
+import fsp from 'node:fs/promises';
+import { execFile, execFileSync } from 'node:child_process';
+import { basename } from 'node:path';
+
+import { makeNodeBundleCache } from '@endo/bundle-source/cache.js';
+import { makeE2ETools } from '../tools/e2e-tools.js';
+import { NonNullish } from '../src/objectTools.js';
+
+const opt0 = {
```

Comment:
I'm unfamiliar with this naming convention. Ok with `defaultOptions` or `defaults`?

Reply:
went away in the move to node's `parseArgs`